src/styles/catalog.scss

```diff
@@ -0,0 +1,13 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  width: var(--card-area-width);
+  margin: 0 auto;
+  padding-top: 50px;
+  padding-left: 40px;
+  
+
+  & :nth-child(4n) {
+    margin-right: 0;
```

Comment:
Setting 'margin-right: 0' on every 4th card might not be sufficient. If you have a margin on card elements, you should also set 'margin-left: 0' on every 1st card of a new row (':nth-child(4n+1)'). This ensures perfect alignment of cards on all rows.

Reply:
Not fixed 
![image](https://github.com/mate-academy/layout_catalog/assets/99403198/300a3221-08c7-42e9-8102-7877df7d7843)

Reply:
@MyNovikov I think you can remove this rule. It should fix the tests also